pkg/kubelet/kubeletconfig/checkpoint/download.go

```diff
@@ -34,6 +34,12 @@ import (
 type RemoteConfigSource interface {
 	// UID returns the UID of the remote config source object
 	UID() string
+	// Name returns the name of the remote config source object
+	Name() string
```

Comment:
I dont think adding `Name()` or `Namespace()` to the api is neccessary.

Reply:
fair enough, removed